applications/launchpad/gui-react/src/containers/SettingsContainer/BaseNodeSettings/BaseNodeSettings.tsx

```diff
@@ -64,6 +81,7 @@ const BaseNodeSettings = ({
           <InputRow>
             <Label $noMargin>{t.baseNode.settings.rootFolder}</Label>
             <Input
+              onClick={() => selectDirectory()}
```

Comment:
... this is the second change:

```suggestion
              onClick={() => selectDirectory(field.value)}
```

Reply:
And done.